UnitySDK/Assets/ML-Agents/Examples/3DBall/Scripts/Ball3DAgent.cs

```diff
@@ -8,10 +8,15 @@ public class Ball3DAgent : Agent
     [Header("Specific to Ball3D")]
     public GameObject ball;
     private Rigidbody ballRb;
+    private ResetParameters resetParams;
+    public float scale;
 
     public override void InitializeAgent()
     {
         ballRb = ball.GetComponent<Rigidbody>();
+        var academy = Object.FindObjectOfType<Academy>() as Academy;
+        resetParams = academy.resetParameters;
```

Comment:
Just checking that this is pass-by-reference, since we're relying on this to be a reference to work across many resets.

Reply:
Yes, I have been testing this. The values seem to match the ones passed into the academy.